### PR TITLE
Added validation for availability of dataset ID

### PR DESCRIPTION
**Pull request by maheshakya**
For scatter plot points and chart sample points.

### CODE REVIEW

File: components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLDatasetProcessor.java [hunk 1]
```diff
@@ -122,6 +122,10 @@ public MLDataset getDataset(int tenantId, String userName, long datasetId) throw
         try {
             List<MLDatasetVersion> versions = databaseService.getAllVersionsetsOfDataset(
                     scatterPlotPoints.getTenantId(), scatterPlotPoints.getUser(), datasetId);
+            // Check whether versions are available for the dataset ID, if not it's not a valid ID
+            if (versions.size() < 1) {
```
Comment (nirmal070125) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLDatasetProcessor.java]: why not equate to 0?

Comment (maheshakya) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLDatasetProcessor.java]: what's the difference?
I was concerning the fact : size should be at least 1 to proceed.

Comment (nirmal070125) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLDatasetProcessor.java]: @maheshakya size can't be negative :) So it's more intuitive to have versions.size == 0  Wdyt?

Comment (maheshakya) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLDatasetProcessor.java]: Makes sense.

**Pull request merged**